### ISSUE TITLE
Data request on multiple instruments with shared variables results in merge error on to_xarray()

### ISSUE THREAD

**Issue by s-kganz** [cormorack/yodapy#48]
Request data from CTD, nitrate instruments on shallow profiler.
```ooi = ooi()
ooi.filter(region="cabled",instrument="S01SBPS-SF01A-2A-CTDPFA102,RS01SBPS-SF01A-4A-NUTNRA101")
```
...
```
dataset_list = ooi.to_xarray()
MergeError: conflicting values for variable 'time' on objects to be combined:
first value: <xarray.Variable (obs: 2505223)>
dask.array<shape=(2505223,), dtype=datetime64[ns], chunksize=(2505223,)>
Attributes:
    long_name:      time
    standard_name:  time
    axis:           T
    _ChunkSizes:    10000
second value: <xarray.Variable (obs: 2505223)>
dask.array<shape=(2505223,), dtype=datetime64[ns], chunksize=(2505223,)>
Attributes:
    long_name:      time
    standard_name:  time
    axis:           T
    _ChunkSizes:    10000
```
Potential fix comes from `xarray.Dataset.merge` documentation:
```
overwrite_vars : str or sequence, optional
    If provided, update variables of these name(s) without checking for conflicts in this dataset.
```
Perhaps identify shared variable names, send to `merge` call and overwrite.

Or, identify shared variable names, append instrument name to each one to differentiate.

**Comment by lsetiawan** [comment]
@s-kganz Yea, `open_mfdataset` uses merge under the hood, I think, so I guess maybe we need to specify how the merge is performed. Thanks for catching this! I will take a look at it.

**Comment by lsetiawan** [comment]
@s-kganz What are the begin and end dates for this request?

**Comment by s-kganz** [comment]
Request done above was from May 1 to May 31, 2016.

**Comment by lsetiawan** [comment]
Resolved in PR #49
